lib/airbrake/utils.ex

```diff
@@ -33,4 +33,27 @@ defmodule Airbrake.Utils do
       do: {k, @filtered_value},
       else: {k, filter(v, filtered_attributes)}
   end
+
+  @doc """
+  Turns tuples into lists for JSON serialization of Airbrake payloads.
+  """
+  def detuple(struct) when is_struct(struct) do
+    struct |> Map.from_struct() |> detuple()
+  end
```

Comment:
So I'd like the function to _just_ de-tuple, not de-struct structs.  (The JSON encoding should de-struct the structs.)  So use this instead:

```
def detuple(%module{} = struct) do
  fields = struct |> Map.from_struct() |> detuple()
  struct(module, fields)
end
```

The test below will also have to change.

This library still supports Elixir 1.10, so we can't use `Kernel.then/2`.

Reply:
Marking as resolved due to needing to drop or Elixir 1.10 support in this PR for completely unrelated reasons (breaking change in 3rd party dependency)